### PR TITLE
Went through and removed all commented out code.

### PR DESCRIPTION
**Pull request by Rovanion**
Hi there lads, I thoroughly enjoy your mod in game and decided to have a look at the code and found two very trivial fixes that I could do in a short amount of time.

First I removed all commented out code with the motivation that the old code is always available in the git history and that having it there makes the code much less comprehensible. Reading code with commented out sections is like reading an article with crossed out paragraphs and sentences.

One thing that I noticed when I removed these dead lines was that a number of catch-clauses didn't do anything. Perhaps it's on purpose, perhaps it's by mistake; but it should be looked in to.

I also removed all trailing white-spaces.

### CODE REVIEW

**Comment by joaofarias** [comment]
Hey Rovanion, thanks for taking a look at it though I won't merge your changes. The reason is that most of the commented out code is not dead code. Some of it is WIP for future releases, some of it is used whenever a new update comes out, some is stuff I have been tinkering with and will come back to later.

I know that WIP should be in different branches but since I'm essentially working alone, I didn't feel the need for it. And I like it much better to simply comment/uncomment code than searching for it in the git history.

Regarding the catch clauses, can you point me to which ones you're talking about and what do you think they should do?

File: CSL-Traffic/Initializer.cs [hunk 1]
```diff
@@ -1810,7 +1687,7 @@ public static void DumpTextureToPNG(Texture previewTexture, string filename = nu
                 }
                 catch (UnityException)
                 {
```
Comment (Rovanion) [inline CSL-Traffic/Initializer.cs:1689]: By searching through the diff for catch you can see the clauses I mention. Here is one.

File: CSL-Traffic/Initializer.cs [hunk 2]
```diff
@@ -1821,23 +1698,19 @@ public static void DumpTextureToPNG(Texture previewTexture, string filename = nu
                         Graphics.Blit(texture, rt);
                         DumpRenderTexture(rt, filename);
                         RenderTexture.ReleaseTemporary(rt);
-                        //Log.Warning(String.Format("Texture dumped to \"{0}\"", filename));
                     }
                     catch (Exception ex)
                     {
-                        //Log.Error("There was an error while dumping the texture - " + ex.Message);
```
Comment (Rovanion) [inline CSL-Traffic/Initializer.cs:1828]: And here's another one.

What you want to do with them is up to you. It requires actual understanding of the code :D

**Comment by Rovanion** [comment]
If I recall correctly there was also one empty if- or else-statements, but I couldn't find it again now when glancing over the diff.

Even if it's only you working on the project I would still recommend you removing code instead of commenting it out if you're removing it from one commit to another one since it's always available in the git repo and removing it makes the code easier for yourself to read.

But it doesn't matter to me, only to you so you do what you feel is right. And cheers mate, thanks for all your hard work. The game is much more enjoyable with it!

**Comment by joaofarias** [comment]
I haven't wrote that code but apparently I forgot to credit it :/
Still, that's only used to extract textures from the game - when I want to extract many at once instead of one by one with modTools - and it never goes in release builds so it's okay.

Thanks for your help anyway :)

**Comment by Rovanion** [comment]
No problem mate!

**Pull request closed without merging**